src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IForEachLoopStatement.vb

```diff
@@ -2926,7 +2926,7 @@ Block[B1] - Block
               Instance Receiver: 
                 IParameterReferenceOperation: x (OperationKind.ParameterReference, Type: C) (Syntax: 'x')
               Arguments(1):
-                  IArgumentOperation (ArgumentKind.Explicit, Matching Parameter: member) (OperationKind.Argument, Type: null, IsImplicit) (Syntax: 'x')
+                  IArgumentOperation (ArgumentKind.DefaultValue, Matching Parameter: member) (OperationKind.Argument, Type: null, IsImplicit) (Syntax: 'x')
```

Comment:
>ArgumentKind.DefaultValue [](start = 38, length = 25)

This change feels unexpected. This entire call is compiler generated, including all the arguments. Otherwise, I would expect the same handling for Current property

Reply:
Yeah, I missed properties. Adding.

---
In reply to: [190035385](https://github.com/dotnet/roslyn/pull/26781#discussion_r190035385) [](ancestors = 190035385,190034922)